BrcmPatchRAM/BlueToolFixup.cpp

```diff
@@ -113,6 +113,7 @@ static bool shouldPatchBoardId = false;
 static bool shouldPatchAddress = false;
 
 static const size_t kBoardIdSize = sizeof("Mac-F60DEB81FF30ACF6");
+static const size_t kBoardIdSizeLegacy = sizeof("Mac-F22586C8");
```

Comment:
How about `constexpr`? The size is fixed anyway.

Reply:
Sounds good, might as well change that while touching the code here. In my defense, I copied the line above 🙂